.gitignore

```diff
@@ -46,3 +46,4 @@ README.html
 
 temp/*
 .konchrc
+.idea
```

Comment:
Files/folders like this belong in a global gitignore: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore